app/components/Button/Button.css

```diff
@@ -1,5 +1,52 @@
 @import url('~app/styles/variables.css');
 
+:root {
```

Comment:
Figured it was best to move this from `variables.css` to here since this is the only file where it is used.